app/_src/gateway/kong-enterprise/workspaces/index.md

```diff
@@ -30,6 +30,20 @@ router:
     * If the matching Service or Route's `host` is an absolute value, a
       conflict is reported—`409 Conflict`
 
+Some entities are "global" entities and thus does not belong to any workspace.
+In particular, the `ca_certificates` is global and used to verify the client
+certificates in mTLS handshaking. The SSL handshaking takes place before receiving
+an HTTP request when the workspace is unknown.
+
+Some entities reside in a workspace but contain fields that should be unique
+across workspaces to avoid conflict. Specifically, the `name` field of the
+`snis` entity should be unique across workspaces because it is used in the
+SSL handshaking phase before the workspace is determined.
```

Comment:
```suggestion
Some entities reside in a workspace but contain fields that should be unique
across workspaces to avoid conflict:
*  The `name` field of the `snis` entity should be unique across workspaces because it is used in the
SSL handshaking phase before the workspace is determined.
```